CS483/CS483/Kartaclysm/Data/UserConfig/ControlBindings.xml

```diff
@@ -1,54 +1,54 @@
 <ControlBindings>
   <Keyboard1>
-      <accelerate value="265" />
-      <brake value="264" />
-      <left value="263" />
-      <right value="262" />
-      <slide value="32" />
-      <driverAbility1 value="65" />
-      <driverAbility2 value="83" />
-      <kartAbility1 value="68" />
+      <accelerate value="87" />
+      <brake value="83" />
+      <left value="65" />
+      <right value="68" />
+      <slide value="90" />
+      <driverAbility1 value="81" />
+      <driverAbility2 value="69" />
+      <kartAbility1 value="82" />
       <kartAbility2 value="70" />
-      <pause value="257" />
+      <pause value="88" />
```

Comment:
The control bindings should be reset back to what they were, or should have MacIntosh sign off on them. If they do stay this way, the hardcoded defaults in InputActionMapping should be changed too.